.github/workflows/build.yml

```diff
@@ -0,0 +1,109 @@
+name: Build
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  ubuntu-gcc:
+    name: Ubuntu 18.04 / gcc
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v2
+    - name: Install build dependencies
+      run: |
+        sudo apt-get update && sudo apt-get install -y --no-install-recommends \
+          ccache \
+          libavformat-dev \
+          libchromaprint-dev \
+          libebur128-dev \
+          libfaad-dev \
+          libfftw3-dev \
+          libflac-dev \
+          libid3tag0-dev \
+          liblilv-dev \
+          libmad0-dev \
+          libmodplug-dev \
+          libmp3lame-dev \
+          libmp4v2-dev \
+          libopus-dev \
+          libopusfile-dev \
+          libportmidi-dev \
+          libprotobuf-dev \
+          libqt5opengl5-dev \
+          libqt5sql5-sqlite \
+          libqt5svg5-dev \
+          libqt5x11extras5-dev \
+          librubberband-dev \
+          libshout3-dev \
+          libsndfile1-dev \
+          libsoundtouch-dev \
+          libsqlite3-dev \
+          libtag1-dev \
+          libupower-glib-dev \
+          libusb-1.0-0-dev \
+          libwavpack-dev \
+          portaudio19-dev \
+          protobuf-compiler \
+          qt5-default \
+          qt5keychain-dev \
+          qtscript5-dev \
+    - name: Set up cmake
+      uses: jwlawson/actions-setup-cmake@v1.4
+      with:
+        cmake-version: '3.13.x'
```

Comment:
What's the reasoning behind the selected version? Is it the minimum required version or the version included in Ubuntu 18.04. Please add a comment about this implicit dependency that also defines when the version needs to be updated.

Reply:
It's the minimum required version defined in our CMakeLists.txt. That should make sure that we don't use functions that are not present in older CMake versions without updating that value.

Reply:
:+1: `# Minimum required version in CMakeLists.txt` ;)